app.js

```diff
@@ -127,16 +136,21 @@ app.post('/food', function(req, res) {
     var q1 = 'INSERT INTO Food (userID, foodType, Description) VALUES (?, ?, ?)';
     var inserts = [session.userID, req.body.foodType, req.body.foodDesc];
     mysql.query(q1, inserts, function(err, result) {
-        if(err) {
-            if (err.code === 'ER_BAD_NULL_ERROR') {
-                context.errorText = "You are not currently logged in. Please go to the login page.";
```

Comment:
Why was this specific error removed?

Reply:
I removed that error to match transportation and because without an intuitive way to go to the login page besides typing into the address bar the location of the landing page, I thought it best not to instruct them to do that.

Reply:
Ideally we should add the login page link to the top of the page, but that can be done in a different PR. Also, by treating a missing user as an error, it spams the server log about errors we shouldn't care about. Only true errors should be printed on the console

Reply:
The login page link may be tricky because it needs to be sensitive to whether a user is logged in or not and we don't have a logout functionality currently. And it's not worth the effort. Given the limited timeframe we have for the project and other classes, I'm not overly concerned with getting everything 100% accurate, just working. I can revert the error for now.

Reply:
But if this belongs on this page, it belongs on every tracking page and it currently isn't on any of them.